File: PHP/Team.php
<?php

class Team {

  private $name;
  private $wins;
  private $loss;

  public function __construct($name, $wins, $loss) {
    $this->name = $name;
    $this->wins = $wins;
    $this->loss = $loss;
  }

  public function getName() {
    return $this->name;
  }

  public function getWins() {
    return $this->wins;
  }

  public function getLosses() {
    return $this->loss;
  }

  public function getWinPercentage() {
    return $this->wins / ($this->wins + $this->loss);
  }

  public function __toString() {
    return $this->name . " (" . $this->wins . ", " . $this->loss . ")";
  }

}

?>

File: src/unl/cse/Team.java
package unl.cse;

public class Team {

	public final String name;
	public final Integer wins;
	public final Integer loss;
	
	public Team(String name, Integer wins, Integer loss) {
		this.name = name;
		this.wins = wins;
		this.loss = loss;
	}
	
	public String getName() {
		return this.name;
	}
	
	public Integer getWins() {
		return this.wins;
	}
	
	public Integer getLoss() {
		return this.loss;
	}
	
	public Double getWinPercentage() {
		return this.wins / (double) (this.wins + this.loss);
	}
	
	public String toString() {
		return this.name + " (" + this.wins + ", " + this.loss + ")";
	}
}
